templates/admin/user/list.tmpl

```diff
@@ -105,6 +106,12 @@
 							{{else}}
 								<td><span>{{ctx.Locale.Tr "admin.users.never_login"}}</span></td>
 							{{end}}
+							<td>
+								<div class="gt-df gt-gap-3">
+									<a href="{{$.Link}}/{{.ID}}" data-tooltip-content="{{ctx.Locale.Tr "admin.users.details"}}">{{svg "octicon-person"}}</a>
```

Comment:
How about also use `{{.HomeLink}}`?

Reply:
It needs `admin` link, but not `home` link.